src/main/java/mik/pet/project/controller/BookController.java

```diff
@@ -23,8 +24,8 @@ public class BookController {
     private BookService bookService;
 
```

Comment:
Please add some details to endpoints for the swagger docs (using dedicated annotations)